### ISSUE TITLE
Norm of a vector is Inf but it should not be

### ISSUE THREAD

**Issue by andreasvarga** [JuliaArrays/StaticArrays.jl#1116]
```
using LinearAlgebra
using StaticArrays
b =    [-4.744752769108950e-01                                                                                                                        
           4.245762257203441e+290]
bs = SA[-4.744752769108950e-01                                                                                                                        
           4.245762257203441e+290]
norm(b)
4.245762257203441e290
norm(bs)
Inf
```
The norms must be same, but are not.

**Comment by hyrodium** [comment]
This is a duplicate of https://github.com/JuliaArrays/StaticArrays.jl/issues/913.

**Comment by andreasvarga** [comment]
I agree, this is a similar case. But then, why is this problem not yet fixed?

**Comment by hyrodium** [comment]
https://github.com/JuliaArrays/StaticArrays.jl/pull/975 will fix this issue, and the PR is just waiting for some reviews.

```julia
julia> using LinearAlgebra, StaticArrays

julia> b =    [-4.744752769108950e-01                                                                                                                        
                  4.245762257203441e+290]
2-element Vector{Float64}:
 -0.474475276910895
  4.245762257203441e290

julia> bs = SA[-4.744752769108950e-01                                                                                                                        
                  4.245762257203441e+290]
2-element SVector{2, Float64} with indices SOneTo(2):
 -0.474475276910895
  4.245762257203441e290

julia> norm(b)
4.245762257203441e290

julia> norm(bs)
4.245762257203441e290
```

I will try making time to review the PR within a week.

**Comment by hyrodium** [comment]
This issue is now fixed in v1.5.13.
https://github.com/JuliaArrays/StaticArrays.jl/releases/tag/v1.5.13
